app/models/group.rb

```diff
@@ -747,7 +747,7 @@ def self.owner_of(groups, user)
 
   def flair_type
     return :icon if flair_icon.present?
-    return :image if flair_upload_id.present?
+    return :image if flair_upload_id.present? && flair_upload.present?
```

Comment:
```suggestion
    return :image if flair_upload.present?
```

Reply:
My idea was to try and avoid a DB query if the flair_upload_id was not present... but in hindsight, `flair_upload` probably does that sanity check anyway 🤦 

Thanks!